fs/fs.go

```diff
@@ -62,6 +63,7 @@ type Context struct {
 	// docker root directory.
 	DockerRoot string
 	DockerInfo map[string]string
+	RktPath    string
```

Comment:
Should this be `RktRoot` for consistency, or does this represent something different from `DockerRoot`?